site/content/installation/running-on-kind.md

```diff
@@ -1,43 +1,61 @@
+- [kind](https://kind.sigs.k8s.io/): Short for _Kubernetes in Docker_, this tool lets you run Kubernetes clusters locally using Docker containers, ideal for testing and development purposes.
 
-Create a cluster with `kind`. You can follow
-their [instructions](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster), or run the following make
-command at the root of the repository:
 
-```makefile
-make create-kind-cluster
-```
+## Create a kind Cluster
+
+To create a kind cluster, choose from the following options:
+
+- **Option 1**: Use the `kind` tool. For detailed instructions, refer to the kind quick start guide [Creating a Cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster).
+
+- **Option 2**: Run the following `make` command in the root of your repository:
+
+   ```makefile
+   make create-kind-cluster
+   ```
+
+   This command creates a kind cluster using the settings from your makefile.
+
 
 ## Deploy NGINX Gateway Fabric
 
-Follow the [installation](./how-to/installation/installation.md) instructions to deploy NGINX Gateway Fabric on your Kind cluster.
+Now that you've created a kind cluster, the next step is to install NGINX Gateway Fabric.
+
+To install NGINX Gateway Fabric, choose the appropriate installation guide that suits your setup:
 
-{{<note>}} For `kind` clusters, NodePort services require [extra configuration](https://kind.sigs.k8s.io/docs/user/configuration/#nodeport-with-port-mappings) and LoadBalancer services need [a third-party controller](https://kind.sigs.k8s.io/docs/user/loadbalancer/) like MetalLB for external IP assignment. However, the Helm chart creates a LoadBalancer service by default. To avoid this, you can disable service creation by adding `--set service.create=false` to your Helm command and use the port-forwarding command below instead to try out the examples. {{</note>}}
+- [Installation with Helm]({{< relref "installation/installing-ngf/helm.md" >}})
+- [Installation with Kubernetes manifests]({{< relref "installation/installing-ngf/manifests.md" >}})
 
-## Access NGINX Gateway Fabric
+{{<note>}}
+When using kind clusters, be aware that NodePort services require [additional setup]((https://kind.sigs.k8s.io/docs/user/configuration/#nodeport-with-port-mappings)). Also, for LoadBalancer services, you’ll need a [third-party controller](https://kind.sigs.k8s.io/docs/user/loadbalancer/) like MetalLB to assign external IPs. The default Helm chart creates a LoadBalancer service; however, you can disable this by adding `--set service.create=false` to your Helm command. Afterward, you can [configure port forwarding](#configure-port-forwarding) as described below to access the examples.
+{{</note>}}
 
-Forward local ports 8080 and 8443 to ports 80 and 443 of the nginx-gateway Pod:
+## Configure Port Forwarding {#configure-port-forwarding}
```

Comment:
Is `{#configuring-port-forwarding}` required by hugo?

Reply:
Not required, but this is how you set a persistent anchor link. If the heading is updated later -- let's say it becomes "Configuring Port Forwarding" -- this anchor won't change and links using it won't break.

Reply:
I like to set persistent anchor links anytime I use one. Like here: 

<img width="886" alt="image" src="https://github.com/nginxinc/nginx-gateway-fabric/assets/33876974/2b436080-5d04-4559-92bc-933c1ed73803">

This way, there's flexibility in updating the heading text w/o breaking the anchor link.